packages/radio-box-group/src/RadioBox.tsx

```diff
@@ -38,35 +38,26 @@ interface StateForStyles {
   size: Size;
 }
 
-const getInteractionRingStyles = ({ checked }: StateForStyles) => {
+const getInteractionRingStyles = (_: StateForStyles) => {
   const baseStyles = css`
     width: 100%;
     height: 100%;
     // Display behind border
     z-index: -1;
   `;
 
-  if (checked) {
-    return cx(
-      baseStyles,
-      css`
-        // Bring in front of border (covering it)
-        z-index: 0;
-      `,
-    );
-  }
-
   return baseStyles;
 };
 
-const getBorderStyles = ({ disabled, size }: StateForStyles) => {
+const getBorderStyles = ({ checked, disabled, size }: StateForStyles) => {
   const baseStyles = cx(
     css`
-      border: 1px solid ${uiColors.gray.base};
+      border: 1px solid ${checked ? 'transparent' : uiColors.gray.base};
```

Comment:
This needs to transition to white to align with the original design. Right now, the border appears 1px thicker than it did before, though the transition is fixed!